app/controllers/concerns/localization.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module Localization
+  extend ActiveSupport::Concern
+
+  included do
+    before_action :set_locale
+
+    protected
+
+    def set_locale
+      I18n.locale = params[:locale] || I18n.default_locale
+    end
+
+    private
+
+    def default_url_options
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/nimblehq/nimble-survey-web/blob/3bdd1603c87059b5dda5aabfb14999fb437bbf29/app/controllers/concerns/localization.rb#L17"></span>Doesn’t depend on instance state (maybe move it to another class?)</td>
    </tr>
  </tbody>
</table>